scripts/searchSamadi.py

```diff
@@ -0,0 +1,20 @@
+import commands, sys
+
+stringToLookFor = "*v0.1.0+76X_HHAnalysis_2016-04-04.v0"
+output = commands.getoutput("search_SAMADhi.py sample -n %s"%stringToLookFor)
```

Comment:
`scripts/*` are added to the `PATH`, while `python/*` are added to `PYTHONPATH`, so you'd need to manually add the path to the python path